src/niweb/apps/scan/migrations/0002_auto_20190121_0856.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this one really part of your changes?

Reply:
Not really, but this migration was on my changes and I don't really recall the moment this has been generated. I've checked the models and the state of the project and seems to me necessary to commit but let me know if it breaks something. This migration sets a default value for the field, which is what the models.py (from scan I think) says, but the previous migrations doesn't state.